style/popup.less

```diff
@@ -24,5 +24,9 @@
 			max-height: unset;
 			top: 0;
 		}
+
+		&.hidden {
```

Comment:
What is this for?

Reply:
Oh good catch, that's leftover from the previous approach so removing...